app/src/main/java/com/github/libretube/util/ImageHelper.kt

```diff
@@ -25,17 +26,24 @@ object ImageHelper {
     fun initializeImageLoader(context: Context) {
         val maxImageCacheSize = PreferenceHelper.getString(
             PreferenceKeys.MAX_IMAGE_CACHE,
-            "128"
-        ).toInt()
-
-        val diskCache = DiskCache.Builder()
-            .directory(context.filesDir.resolve("coil"))
-            .maxSizeBytes(maxImageCacheSize * 1024 * 1024L)
-            .build()
+            ""
+        )
 
         imageLoader = ImageLoader.Builder(context)
             .callFactory(CronetHelper.callFactory)
-            .diskCache(diskCache)
+            .apply {
+                when (maxImageCacheSize) {
+                    "" -> {
+                        memoryCachePolicy(CachePolicy.DISABLED)
+                    }
+                    else -> diskCache(
```

Comment:
Would this mean caching is disabled by default?

Reply:
Yes, is that a problem for the servers?
I'm always annoyed that the app size increases so much :/

Reply:
No, it's not a problem for servers but is a problem for app performance.
I think we should use both a memory and disk cache by default.

Let's say 20MB for disk, and 5MB of ram based caching.